java/src/main/java/org/rocksdb/OptimisticTransactionDB.java

```diff
@@ -212,6 +233,15 @@ public Transaction beginTransaction(final WriteOptions writeOptions,
     return oldTransaction;
   }
 
+  /**
+   * Get the underlying database that was opened.
+   *
+   * @return The underlying database that was opened.
+   */
+  OccValidationPolicy getOccValidationPolicy() {
```

Comment:
Function scope should be "public" not "package-private"

Reply:
Getters in RocksJava are typically not prefixed `get` - see other classes for the naming convention.